lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -290,15 +294,19 @@ sub limit_assets {
         delete $toremove{$id};
     }
     if ($doremove) {
-        my $removes = $app->db->resultset('Assets')->search({id => {in => [sort keys %toremove]}}, {order_by => qw/t_created/});
+        my $removes
+          = $app->db->resultset('Assets')->search({id => {in => [sort keys %toremove]}}, {order_by => qw/t_created/});
         while (my $a = $removes->next) {
             $a->remove_from_disk;
             $a->delete;
         }
     }
     my $timecond = {"<" => time2str('%Y-%m-%d %H:%M:%S', time - 24 * 3600, 'UTC')};
 
-    my $assets = $app->db->resultset('Assets')->search({t_created => $timecond, type => [qw(iso hdd repo)], id => {-not_in => [sort keys %seen_asset]}}, {order_by => [qw(t_created)]});
+    my $assets
+      = $app->db->resultset('Assets')
+      ->search({t_created => $timecond, type => [qw(iso hdd repo)], id => {-not_in => [sort keys %seen_asset]}},
+        {order_by => [qw(t_created)]});
```

Comment:
Normally my formatting strategy for a multi-line expression like this would be to split it up into separate expressions. Which tends to not affect performance too much and works rather well with perltidy.
```perl
    my $search = {t_created => $timecond, type => [qw(iso hdd repo)], id => {-not_in => [sort keys %seen_asset]}};
    my $assets = $app->db->resultset('Assets')->search($search, {order_by => [qw(t_created)]});
```
Do we want to do that in openQA too?